cpg_workflows/scripts/talos_prep/extract_fragmented_vcf_from_mt.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+"""
+Takes a MatrixTable and two output paths
+Writes two representations - region filtered MatrixTable, and a sites-only VCF representation
+VCF is exported per-interval, with a separate header file, to be concatenated later
+
+This script also takes a BED file as input; output contains only the variants that overlap with the BED file
+
+All existing info fields are dropped, and replaced with just the callset AC / AN / AF
```

Comment:
For the external pipeline we may need to reconsider this in the long run. There is a chance people will want to provide a vcf that includes the equivalent of our callset AN/AF etc. We may need to support a pathway where we can take those and use them for our callset filtering operations.

Reply:
Yup, that makes sense. It'll be interesting to see what different sites have a data source for that... whether it's annotation hopped from a VCF, or from a completely different data source

Reply:
Yea, hard to predict. Simon maintains some form of observed variants database that he can annotate his fam specific vcfs with for this purpose.